osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -89,17 +91,76 @@ private void setDistances(double clockRate)
             if (lastObject is Slider lastSlider)
             {
                 computeSliderCursorPosition(lastSlider);
-                TravelDistance = lastSlider.LazyTravelDistance * scalingFactor;
+                TravelDistance = 0;
                 TravelTime = Math.Max(lastSlider.LazyTravelTime / clockRate, min_delta_time);
                 MovementTime = Math.Max(StrainTime - TravelTime, min_delta_time);
+                MovementDistance = Vector2.Subtract(lastSlider.TailCircle.StackedPosition, BaseObject.StackedPosition).Length * scalingFactor;
+
+                int repeatCount = 0;
+
+                Vector2 currSliderPosition = ((OsuHitObject)lastSlider.NestedHitObjects[0]).StackedPosition;
+
+                for (int i = 1; i < lastSlider.NestedHitObjects.Count; i++)
+                {
+                    var currSliderObj = (OsuHitObject)lastSlider.NestedHitObjects[i];
+
+                    Vector2 currSlider = Vector2.Subtract(currSliderObj.StackedPosition, currSliderPosition);
```

Comment:
This took me a bit to realize what this means, `currSlider` isn't exactly an ideal variable name here. Thoughts on something like `currNestedToPrev` or something similar?

(similar issue with `lazySlider` on 115, not super descriptive)

Reply:
rewritten to be currMovement and lazyMovement